plugin/completion.py

```diff
@@ -24,25 +24,25 @@ def run(self, edit: Any, item: dict) -> None:
 
             range = Range.from_lsp(text_edit['range'])
             edit_region = range_to_region(range, self.view)
+            # subtract the typed prefix form the end of the text edit
+            edit_region = sublime.Region(edit_region.begin(), edit_region.end() - len(CompletionHandler.last_prefix))
```

Comment:
Can it happen that `edit_region.begin() > edit_region.end() - len(CompletionHandler.last_prefix)` ?

Reply:
It can or should I say it could (I removed that logic)
I concluded that is best to NOT touch the TextEdit Range. 
And instead when the user select a command completion item.
the prefix gets removed (I cannot stop that behavior, thats how sublime works...)
so the text edit range is not valid anymore(and I need to calculate the offset, which from my point of view now, seems impossible)

But the solutions is simple.
I just insert back the removed prefix, before continuing with the insert of the TextEdit 
which makes the text edit range valid again. 

If it all makes sense, if not I can try to explain it better.